test/test_helper.rb

```diff
@@ -18,7 +18,7 @@ def exists?
 class ActionDispatch::IntegrationTest
   include Rails.application.routes.url_helpers
 
-  Capybara.default_max_wait_time = 10
+  Capybara.default_max_wait_time = 60
```

Comment:
I saw this too, but this is the wrong timeout. But I can't find any recent documentation for the right timeout. See:

https://gist.github.com/mikegehard/922296?permalink_comment_id=901298#gistcomment-901298

Reply:
I tried this and it didn't fix it. I'm not sure if it's the right way to set the timeout though since comments I found mentioning this were 6+ years old:
https://github.com/AllYourBot/hostedgpt/pull/483/files#diff-ba37813ca277c227a74a372479b7b05b7f3ff085d890ab708f80d62573efdb7a

Reply:
I think I was able to reproduce the timeout last night using act and running the workflow locally. Will keep investigating today, but wanted to mention that. Not sure if you're using "act" already, but it's nice if you don't want to have to push changes everyone every time you want to test something: https://github.com/nektos/act

Reply:
That's a great find! It didn't even occur to me to try running GH actions locally.

My best thought from earlier investigation was that there probably is indeed some timeout variable deep in capybara we could set to make this go away, what I tried may simply be outdated since the comments were old. It might take inspecting the source of the right gem (copyabara, selenium, rails? not sure which).

But my one new thought  is that the application file loads some assets remotely (from cdn) and when I was on a plane last with really flaky internet the system tests seem to hang forever while trying to request those (although I didn't get that error). I wonder if there's some internal GH action network flakiness which is causing those asset requests to hang. We may need to download those files and add them to the repo.